pkg/anago/release.go

```diff
@@ -451,6 +460,52 @@ func (d *DefaultRelease) CreateAnnouncement() error {
 	return nil
 }
 
+// UpdateGitHubPage Update the GitHub release page, uploading the
+// source code
+func (d *DefaultRelease) UpdateGitHubPage() error {
+	// Array of assets to be published in the release page
+	assetList := []string{
+		// Build the path to the kubernetes tar file:
+		filepath.Join(
+			gitRoot, // /workspace/src/k8s.io/kubernetes
+			fmt.Sprintf("%s-%s", release.BuildDir, d.state.versions.Prime()), // _output-v1.20.0-beta.3/
+			release.GCSStagePath,     // gcs-stage/
+			d.state.versions.Prime(), // v1.20.0-beta.3
+			release.KubernetesTar,    // kubernetes.tar.gz
+		) + ":Kubernetes Source Code",
+	}
+
+	// URL to the changelog:
+	changelogURL := fmt.Sprintf(
+		"https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-%d.%d.md",
+		d.state.semverBuildVersion.Major,
+		d.state.semverBuildVersion.Minor,
+	)
+
+	// Build the options set for the GitHub page
+	ghPageOpts := &announce.GitHubPageOptions{
+		AssetFiles:            assetList,
+		Tag:                   d.state.versions.Prime(),
+		NoMock:                d.options.NoMock,
+		UpdateIfReleaseExists: true,
+		Name:                  "Kubernetes " + d.state.versions.Prime(),
+		Draft:                 false,
+		Owner:                 git.DefaultGithubOrg,
+		Repo:                  git.DefaultGithubRepo,
+		// PageTemplate: ,     // If we use a custom template, define it here
```

Comment:
Do we mean to leave this comment in?

Reply:
Oh yes I left it intentionally as a guide when we decide what should become of the page  ([ref this in slack](https://kubernetes.slack.com/archives/CJH2GBF7Y/p1606280899195000?thread_ts=1606254574.186400&cid=CJH2GBF7Y))
But I can take it out if it's better :)